src/app/organisations/views/confirmEditOrganisation.ejs

```diff
@@ -0,0 +1,37 @@
+<a href="edit-organisation" class="link-back">Back</a>
```

Comment:
a backLink variable of `edit-organisation` should mean we don't need this as the backLink is defined in the template layout

Reply:
I also just realised that you're using the older style and any new pages going forward need to be using the newer styling.
Fixing it is a little bit of a hassle but it's not too burdensome.  If you need to add the new `layout` to be the new one like https://github.com/DFE-Digital/login.dfe.support/blob/main/src/app/organisations/getCreateOrganisation.js#L6

Then you can use https://design-system.service.gov.uk/ (and/or just current examples in this repo that also use the new styles) to display the page.